hamocc/mo_apply_rivin.F90

```diff
@@ -94,46 +97,64 @@ subroutine apply_rivin(kpie,kpje,kpke,pddpo,omask,rivin)
           enddo
 
           if (use_cisonew) then
-            ocetra(i,j,1:kmle(i,j),isco213)    = ocetra(i,j,1:kmle(i,j),isco213)    +                   &
-                 ocetra(i,j,1:kmle(i,j),isco213)/(ocetra(i,j,1:kmle(i,j),isco212)+safediv)*             &
-                 (rivin(i,j,iralk)*fdt/volij + rivin(i,j,irdin)*fdt/volij + rivin(i,j,irdip)*fdt/volij)
-            ocetra(i,j,1:kmle(i,j),isco214)    = ocetra(i,j,1:kmle(i,j),isco214)    +                   &
-                 ocetra(i,j,1:kmle(i,j),isco214)/(ocetra(i,j,1:kmle(i,j),isco212)+safediv)*             &
-                 (rivin(i,j,iralk)*fdt/volij + rivin(i,j,irdin)*fdt/volij + rivin(i,j,irdip)*fdt/volij)
-
-            ocetra(i,j,1:kmle(i,j),idoc13)     = ocetra(i,j,1:kmle(i,j),idoc13)     +                   &
-                 ocetra(i,j,1:kmle(i,j),idoc13)/(ocetra(i,j,1:kmle(i,j),idoc)+safediv)*                 &
-                 rivin(i,j,irdoc)*fdt/volij
-            ocetra(i,j,1:kmle(i,j),idoc14)     = ocetra(i,j,1:kmle(i,j),idoc14)     +                   &
-                 ocetra(i,j,1:kmle(i,j),idoc14)/(ocetra(i,j,1:kmle(i,j),idoc)+safediv)*                 &
-                 rivin(i,j,irdoc)*fdt/volij
-
-            ocetra(i,j,1:kmle(i,j),idet13)     = ocetra(i,j,1:kmle(i,j),idet13)     +                   &
-                 ocetra(i,j,1:kmle(i,j),idet13)/(ocetra(i,j,1:kmle(i,j),idet)+safediv)*                 &
-                 rivin(i,j,irdet)*fdt/volij
-            ocetra(i,j,1:kmle(i,j),idet14)     = ocetra(i,j,1:kmle(i,j),idet14)     +                   &
-                 ocetra(i,j,1:kmle(i,j),idet14)/(ocetra(i,j,1:kmle(i,j),idet)+safediv)*                 &
-                 rivin(i,j,irdet)*fdt/volij
+            ocetra(i,j,1:kmle(i,j),isco213) = ocetra(i,j,1:kmle(i,j),isco213) +                    &
```

Comment:
Hi @JorgSchwinger and @TomasTorsvik , here and throughout the document: when we do line continuation: what is good practice in terms of where to put the math symbol? - thus far, I personally preferred to have it in the next line, as in (pseudo-code):
```
ocetra = ocetra*xxx &
               + yyy
```
to know, what is happening with the stuff in the continuation line. But is there common agreement on this? - at the moment, I see a mixed approach... - not sure, if we should change this, though.

Reply:
I don't have a strong opinion on this, but I tend to put the math operator first on a new line and not last on the preceding line.

Reply:
I will change this here (first in the new line), since it is a mix here, but I won't go through the code.